activity.h

```diff
@@ -94,9 +94,35 @@ struct PolluteDataCacheConfig {
   int array_reads_per_iteration;
 };
 
+// Activity: PolluteInstructionCache
+
+typedef int (*MyFunctionPtr)(bool);
+class PolluteInstructionCache : public Activity {
+ public:
+  static absl::Status ValidateConfig(ActivityConfig& ac);
+  void Initialize(ParsedActivityConfig* config) override;
+  void DoActivity() override;
+  ActivityLog GetActivityLog() override;
+
+ private:
+  int iteration_count_ = 0;
+  int array_reads_per_iteration_ = 0;
+  int max_func_num_ = 0;
+  std::mt19937 mersenne_twister_prng_;
+  std::vector<MyFunctionPtr> func_ptr_array_;
+  int64_t optimization_preventer_ = 0;
+  std::uniform_int_distribution<> random_index_;
+};
+
+struct PolluteInstructionCacheConfig {
+  int array_reads_per_iteration;
```

Comment:
array_reads is a poort name for this variable. Functions invoked maybe?

Reply:
changed it to - function_invocations_per_iteration